app/controllers/hyku/invitations_controller.rb

```diff
@@ -2,32 +2,31 @@
 
 module Hyku
   class InvitationsController < Devise::InvitationsController
-    # For devise_invitable, specify post-invite path to be 'Manage Users' form
-    # (as the user invitation form is also on that page)
-    def after_invite_path_for(_resource)
-      hyrax.admin_users_path
-    end
+      # For devise_invitable, specify post-invite path to be 'Manage Users' form
```

Comment:
the spacing looks off to me in this file

Reply:
was this just linting on this file? I don't recall it being a part of the theming work?